carLot.js

```diff
@@ -0,0 +1,54 @@
+var CarLot = (function () {
+  var inventory = [];
+
+  return {
+
+    getInventory: function () {
+      return inventory;
+    },
+
+    loadInventory: function (cb) {
+      var inventoryLoader = new XMLHttpRequest();
+
+      inventoryLoader.open('GET', 'inventory.json');
+      inventoryLoader.addEventListener("load", function (evt) {
+        inventory = JSON.parse(evt.target.responseText).cars;
+        cb(inventory);
+      });
+      inventoryLoader.send();
+    },
+
+    activateEvents: function() {
+      var carBlock = document.querySelectorAll('.car-block');
+      var input = document.querySelector('input');
+      var descript;
+
+      carBlock.forEach(e => e.addEventListener('click', changeBlock));
+      input.addEventListener('keypress', updateDescription);
+
+      function changeBlock(evt){
+        document.querySelector('input').removeEventListener('keyup', changeDescript);
+        document.querySelectorAll('.car-block').forEach(e => e.style = `border: 3px solid ${e.style.borderTopColor}`);
+
+        this.style.border = `5px solid ${evt.currentTarget.style.borderTopColor}`;
+        this.style.background = "#e6e6e6";
+        input.focus();
+        input.value = "";
+
+
+        descript = evt.currentTarget.childNodes[11];
```

Comment:
I am going to try to mess with this tonight before merging.  Will see if I can figure it out.